server/src/test/java/org/elasticsearch/index/query/SpanMultiTermQueryBuilderTests.java

```diff
@@ -246,7 +246,7 @@ public void testFromJson() throws IOException {
         assertEquals(json, 1.08, parsed.innerQuery().boost(), 0.0001);
     }
 
-    public void testDefaultMaxRewriteBuilder() throws Exception {
+    public void testDefaultMaxInterceptAndRewriteBuilder() throws Exception {
```

Comment:
That seems unrelated

Reply:
Oh hey, thanks IntelliJ for the refactor/rename errors 🤦